src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
we don't need to extend ColorSupplier just to use its static method like ColorSupplier.getRandomColor().
1st: it is not logic to extend class ColorSupplier by class Lotery. just think
2nd: as i said, u can just call method ColorSupplier.getRandomColor() without extending.

```suggestion
public class Lottery {
```